schema/fields/date.js

```diff
@@ -2,25 +2,39 @@ import moment from 'moment';
 import * as tz from 'moment-timezone'; // eslint-disable-line no-unused-vars
 import { GraphQLString, GraphQLBoolean } from 'graphql';
 
+export function date(rawDate, format, timezone) {
+  if (timezone) {
+    if (format) return moment(rawDate).tz(timezone).format(format);
+    return moment(rawDate).tz(timezone).format();
+  }
+  if (format) return moment.utc(rawDate).format(format);
+  return rawDate;
+}
+
 export default {
   type: GraphQLString,
   args: {
     format: {
       type: GraphQLString,
     },
+    // Accepts a tz database timezone string. See http://www.iana.org/time-zones,
+    // https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
     timezone: {
       type: GraphQLString,
+      description: 'Specify a tz datbase time zone, otherwise falls back to the timezone set in `X-TIMEZONE` header',
```

Comment:
sorry what? is there really no multi-line string support? I just have to make the description shorter? 

`Multiline support is limited to browsers supporting ES5 only  no-multi-str`